com.haysell.pos.metainfo.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Karapet Khachoyan -->
+<component type="desktop-application">
```

Comment:
This is missing multiple necessary tags. Validate the file with https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation and add them

Reply:
this issue was fixed.
Also this file copied to [upstream](https://github.com/Delphi251189/haysell-pos/blob/main/com.haysell.pos.metainfo.xml)